backend/tracim_backend/tests/functional/test_elasticsearch_search.py

```diff
@@ -138,6 +138,144 @@ def test_api___elasticsearch_search_ok__by_filename(
         if first_content_name:
             assert search_result["contents"][0]["label"] == first_content_name
 
+    @parameterized.expand(
+        [
+            # content_name, search_string, nb_content_result, first_content_name, content_body
+            # exact syntax
+            ("good practices", "texttosearch", 1, "good practices", "texttosearch")
+        ]
+    )
+    def test_api___elasticsearch_search_ok__by_description(
+        self, content_name, search_string, nb_content_result, first_content_name, content_body
```

Comment:
as discussed, maybe rename:
- rename content_name to created_content_name
- rename first_content_name to first_result_content_nale
- move content_body to second position and rename it to created_content_body
- for search_string, use a subpart of the content_body in order to make the test case more clear. Eg content_body is "I have fixed a bug" and search string is "bug"

Reply:
done.